chardinjs.js

```diff
@@ -139,7 +147,7 @@
 
                 this.$el.get()[0].appendChild(overlay_layer);
 
-                this.$el.find("#chardin-mask").fadeIn();
+                this.$el.find("#chardinjs-mask").fadeIn();
```

Comment:
#chardin-mask?

Reply:
So element IDs should follow the data attribute convention "chardin" and not the css class convention "chardinjs"?